version.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+"""opendatahub operator."""
+
+
+__version__ = "0.8.0"
```

Comment:
Is the resulting tag going to be `v0.8.0` based on this or just `0.8.0`?

Reply:
its going to be based on v0.8.0 , for example 
if we do a patch release, the release would be for v0.8.1
it would be update in here as well, after the process is finished.

i saw this was the biggest version available in here: https://quay.io/repository/opendatahub/opendatahub-operator?tab=tags
so initiated with it.

Reply:
Sounds good! One issue we might have though is that we do not have 0.8.0 branch at the moment, so IIUC when we merge this the 0.8.0 tag would be built based on master?

Reply:
oh no, the tag would not be built on this. 
when someone opens an issues like `new patch release` , then kebechet would build the branch with the changelog then based on that other bot sefkhet-abwy will get the tag in , and then aicoe-ci will create the image based on that.

Reply:
Just one minor bit - the bot will not find older release and will compute changelog from the beginning of the git log. We could warn about this fact in the opened PR.